ros2_control_demo_description/rrbot_description/ros2_control/rrbot_transmissions_system_position_only.ros2_control.xacro

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+  <xacro:macro name="rrbot_transmissions_system_position_only" params="name prefix use_gazebo:=^|false use_fake_hardware:=^|true mock_sensor_commands:=^|false slowdown:=2.0">
```

Comment:
we can further simplify this by deleting `use_gazebo`, `use_fake_hardware`, `mock_sensor_comands` here

Reply:
Unfortunately this breaks the launch, since we use `rrbot_base.launch.py` as base. Maybe something that could be addressed in #231.

Reply:
does it? removing them here and lines 34+35 in rrbot_transmissions_system_position_only.urdf.xacro works for me.

Reply:
in the interest of merging this prior to the restructure, I'll open a general cleanup issue for xacro parameters

https://github.com/ros-controls/ros2_control_demos/issues/240